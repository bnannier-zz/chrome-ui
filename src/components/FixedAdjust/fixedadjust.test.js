import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });

import { FixedAdjust } from "../";

describe("FixedAdjust Tests", () => {
  it("Default", function() {
    const wrapper = shallow(<FixedAdjust>test</FixedAdjust>);
    expect(wrapper).toMatchSnapshot();
  });
});
