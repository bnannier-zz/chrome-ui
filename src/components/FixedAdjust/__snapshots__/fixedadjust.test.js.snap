// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FixedAdjust Tests Default 1`] = `
<Base>
  <div
    className={
      Array [
        Object {
          "mdc-top-app-bar--dense-fixed-adjust": undefined,
          "mdc-top-app-bar--fixed-adjust": undefined,
          "mdc-top-app-bar--prominent-fixed-adjust": undefined,
          "mdc-top-app-bar--short-fixed-adjust": undefined,
        },
      ]
    }
  >
    test
  </div>
</Base>
`;
